From 183a27bfe59bd852c61b6cd9cfa4dda1f7e7b4d4 Mon Sep 17 00:00:00 2001
From: Emil Medve <Emilian.Medve@Freescale.com>
Date: Wed, 8 Jan 2014 04:41:02 -0600
Subject: [PATCH 0655/1089] fsl_qbman: Fix the checkpatch warning type
 STATIC_CONST_CHAR_ARRAY

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

WARNING:STATIC_CONST_CHAR_ARRAY: static const char * array should probably be static const char * const

Signed-off-by: Emil Medve <Emilian.Medve@Freescale.com>
Change-Id: I7cc81cbcf8ea5ce286111c2ab0f407d6d4d68bd1
Reviewed-on: http://git.am.freescale.net:8181/7787
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Haiying Wang <Haiying.Wang@freescale.com>
Reviewed-by: Geoff Thorpe <Geoff.Thorpe@freescale.com>
Reviewed-by: Emilian Medve <Emilian.Medve@freescale.com>
---
 drivers/staging/fsl_qbman/qman_debugfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl_qbman/qman_debugfs.c b/drivers/staging/fsl_qbman/qman_debugfs.c
index 4e029d6..07a2435 100644
--- a/drivers/staging/fsl_qbman/qman_debugfs.c
+++ b/drivers/staging/fsl_qbman/qman_debugfs.c
@@ -38,7 +38,7 @@ static u32 fqid_max;
 static u64 qman_ccsr_start;
 static u64 qman_ccsr_size;
 
-static const char *state_txt[] = {
+static const char * const state_txt[] = {
 	"Out of Service",
 	"Retired",
 	"Tentatively Scheduled",
-- 
2.0.2

